src/client/components/meal/meal.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from "react";
+import "./meal.css";
+const Meal = () => {
+  const [meal, setmeal] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  useEffect(() => {
+    (async () => {
```

Comment:
There is not much need to make this a function and call it instantly. I can be just actions inside the use effect callback, and the callback can be `async` 👌🏽

Reply:
Hi Karo
Thanks for your valuable feedback,actually this self invoked syntax we have been taught in that week class sessipn and so I thought to apply here.
From next time I will try to avoid this